.github/workflows/main.yml

```diff
@@ -55,23 +62,32 @@ jobs:
         if: ${{ matrix.test_type == 'NPM' }}
         env:
           S3_ENCRYPT_KEY: ${{ secrets.S3_ENCRYPT_KEY }}
-          TRAVIS_JOB_ID: ${{ github.run_number }}
+          # The need for this old environment variable name will go away soon.
+          # Once the new snovault is in, we should retain these for a little while to make sure the right
+          # variable is being used, and then this can go away. -kmp 9-Mar-2021
+          # I have named the ID slightly differently so we can see if the old names are in use.
+          TRAVIS_JOB_ID: ff-npm-x-test-${{ github.run_number }}-
+          # This will be the new environment variable name.
+          TEST_JOB_ID: ff-npm-test-${{ github.run_number }}-
         run: |
           npm test
           make travis-test-npm
+          poetry run wipe-test-indices $TEST_JOB_ID search-fourfront-testing-6-8-kncqa2za2r43563rkcmsvgn2fq.us-east-1.es.amazonaws.com:443
+          # Until the next version of snovault, the following two are prudent. We can remove them soon. -kmp 9-Mar-2021
+          poetry run wipe-test-indices $TRAVIS_JOB_ID search-fourfront-testing-6-8-kncqa2za2r43563rkcmsvgn2fq.us-east-1.es.amazonaws.com:443
+          poetry run wipe-test-indices ff-$TRAVIS_JOB_ID search-fourfront-testing-6-8-kncqa2za2r43563rkcmsvgn2fq.us-east-1.es.amazonaws.com:443
```

Comment:
Only one is strictly needed. But which? Heh. Yeah, we didn't talk about this maybe in so much detail but here is the reason:

* The Travis ones are there only temporarily so that if something is looking at TRAVIS_ID still because it's getting an old copy of the library, it will be there.  

* The value is different so when we see it in the index on Kibana, we'll know it's picking the wrong one. But that's better than picking none.

* The extra ff- prefix is because in the old scheme it used to add that prefix behind your back and you were having trouble deleting them. It also in that circumstance adds -test- at the end, so in fact the index it's looking to delete in that case is probably 'ff-ff-test-123--test-' (assuming I didn't screw up the logic) but just looking for 'ff-ff-test-123-' at the start is enough. We won't be deleting something we care about, so it seems harmless.

Both TRAVIS ids can go away when we're confident that (a) latest utils is on fourfront and (b) we're not doubting something in recent history that would make us unroll the latest version. I think that I'd leave it about a month. It's not pretty, but it'll keep us from failing to delete indexes if something is amiss.

Defensive programming, they call it. :)

Reply:
Oh, except just to be sure, since you made me doubt myself, I went and checked snovault. In fact, it thinks you might give it a number so it turns it into ff-test-%s- which means I need to look for a ff-test- in front. This will probably never get used, but it's a weekend and letting it test one extra time is cheap, so I'm fixing it to soothe my wounded pride.